src/task.rs

```diff
@@ -58,17 +68,37 @@ pub struct TaskConf {
 	pub env: Vec<String>,
 }
 
+fn reset_signals() {
+    let nb_signals: i32 = 39;
+    for n in 1..nb_signals {
+        unsafe { signal(n, SIG_DFL); }
+    }
+}
+
 impl TaskConf {
 	pub fn run(&self) -> i32 { // TODO: handle errors
 		match unsafe { fork() } {
 			pid if pid > 0 && pid < INT_MAX => pid,
 			_ => {
+                                let stdout: File = OpenOptions::new().write(true).truncate(true).create(true).open(&self.stdout.as_path()).unwrap();
+                                let stderr: File = OpenOptions::new().write(true).truncate(true).create(true).open(&self.stderr.as_path()).unwrap();
```

Comment:
```suggestion
                                let stdout: File = OpenOptions::new().write(true).create(true).open(&self.stdout.as_path()).unwrap();
                                let stderr: File = OpenOptions::new().write(true).create(true).open(&self.stderr.as_path()).unwrap();
```
Pas necessaire de truncate